src/util/events.js

```diff
@@ -8,7 +8,14 @@ export function getFilteredEvents(events, filters, zipcodes) {
     return events;
   }
 
-  const zipCodesLength = (Object.keys(zipcodes) || []).length;
+  const zipcodesLength = (Object.keys(zipcodes) || []).length;
+  const validZipcode = zipcodesLength && !!zipcodes[filters.zipcode];
+
+  // We do not yet have zipcodes to filter against, or the selected
+  // zipcode is not valid...so there can be no valid events. Bail early.
+  if (filters.zipcode && (!zipcodes || !zipcodesLength || !validZipcode)) {
```

Comment:
The use case is: user loads page with a zipcode in the query string, component tries to get events based on zipcode, but either zipcodes haven't loaded yet or they have and the selected zipcode is invalid. I can't remember how all this is wired up; will think about it more...